### PR TITLE
doc: add governance for LFX Crowdfunding account

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: Nodejs-Bug-Bounty-Security-Fund.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Node.js Bug Bounty/Security Fund
+
+The TSC maintains a LFX Crowdfunding account in order to accept
```
Comment (Trott) [inline Nodejs-Bug-Bounty-Security-Fund.md:3]: Will people know what LFX is? (I don't, unless it's Linux Foundation, in which case maybe we should spell that out?)

Comment (Trott) [inline Nodejs-Bug-Bounty-Security-Fund.md:3]:
```suggestion
The TSC maintains a LFX Crowdfunding account to accept
```

File: Nodejs-Bug-Bounty-Security-Fund.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+the project component of bug bounties paid out by the
```
Comment (Trott) [inline Nodejs-Bug-Bounty-Security-Fund.md:4]:
```suggestion
the project's share of bug bounties paid out by the
```

File: Nodejs-Bug-Bounty-Security-Fund.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+The funds in the account are intended to be used by the TSC
+to encourage contributions to the project with respect to
+security and in particular the handling of vulnearabilities.
+
+This document outlines the use this account.
+
+## Account details
+
+The [account](https://crowdfunding.lfx.linuxfoundation.org/initiative/578a541a-4e7e-47a2-99b9-6cbf49b00c20)
+is associated with and LFX account. The user id and password
```
Comment (Trott) [inline Nodejs-Bug-Bounty-Security-Fund.md:16]: "is associated with and LFX account" Is that supposed to be "an LFX"?  You use "a LFX" above. Pick one and stick with it.

File: Nodejs-Bug-Bounty-Security-Fund.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+is shared with TSC members through 1password.
```
Comment (Trott) [inline Nodejs-Bug-Bounty-Security-Fund.md:17]:
```suggestion
are shared with TSC members through 1password.
```

File: Nodejs-Bug-Bounty-Security-Fund.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Node.js Bug Bounty/Security Fund
+
+The TSC maintains a LFX Crowdfunding account in order to accept
+the project component of bug bounties paid out by the
+[Hackerone](https://www.hackerone.com/) bug bounty program.
+
+The funds in the account are intended to be used by the TSC
```
Comment (Trott) [inline Nodejs-Bug-Bounty-Security-Fund.md:7]:
```suggestion
The funds are to be used
```

File: Nodejs-Bug-Bounty-Security-Fund.md (hunk 4 above)
Comment (targos) [reply Nodejs-Bug-Bounty-Security-Fund.md:17]: Is it shared with a subset of TSC members? I don't have access to any Node.js-related 1Password vault.

**Review by mcollina — APPROVED** [review]
lgtm

**Review by jasnell — APPROVED** [review]

**Review by legendecas — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

File: Nodejs-Bug-Bounty-Security-Fund.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# Node.js Bug Bounty/Security Fund
+
+The TSC maintains a LFX Crowdfunding account to accept
```
Comment (mhdawson) [inline Nodejs-Bug-Bounty-Security-Fund.md:3]:
```suggestion
The TSC maintains a Linux Foundation Crowdfunding account to accept
```

File: Nodejs-Bug-Bounty-Security-Fund.md (hunk 3 above)
Comment (mhdawson) [reply Nodejs-Bug-Bounty-Security-Fund.md:16]:
```suggestion
is associated with and Linux Foundation Crowdfunding account. The user id and password
```

File: Nodejs-Bug-Bounty-Security-Fund.md (hunk 1 above)
Comment (mhdawson) [reply Nodejs-Bug-Bounty-Security-Fund.md]: expanded

**Pull request merged**